src/ucs/algorithm/string_distance.c

```diff
@@ -0,0 +1,49 @@
+/**
+* Copyright (C) NVIDIA 2022.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "string_distance.h"
+#include <ucs/sys/math.h>
+#include <ucs/sys/sys.h>
+#include <string.h>
+
+#define MAX_STRING_LENGTH 50

```

Comment:
I think the name is too common and may be already defined in some other headers (even in system ones). So let's make it more targeted.
```suggestion
#define UCS_STRING_DISTANCE_MAX_LENGHT 50
```